### PR TITLE
Chore/Routing-Configuration

### PR DESCRIPTION
**Pull request by krishnakpandian**
…ng getAccessTokenSilently and emailVerified

Problem
=======
What problem or issue does this pull request solve [Jira](https://cruzhacks-dev.atlassian.net/jira/software/c/projects/C2D/issues/)

* All Views need to have some Unit Tests to ensure functionality
* AccessTokens need to be tested

Solution
========
What I/we did to solve this problem
* Added unit tests and stylesheets to each View
* Added additional Validation to portal Route to retrieve Access token

Change Summary:
---------------
* Adding all tests for each route, Setting App as main container, Testing getAccessTokenSilently and emailVerified

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* Api.ts and env is for testing purposes only

### CODE REVIEW

File: src/components/Debug/DebugBar.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const DebugBar: React.FC = () => {
       <NavLink exact to='/' className='link' activeClassName='active'>
         Main
       </NavLink>
-      <NavLink exact to='home' className='link' activeClassName='active'>
+      <NavLink exact to='portal' className='link' activeClassName='active'>
         Home
```
Comment (w-gao) [inline src/components/Debug/DebugBar.tsx]:
```suggestion
        Portal
```

File: src/views/Error/index.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import ErrorView from './index.view';
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Testing Error View', () => {
+  it('Error Page renders without crashing', () => {
+    shallow(<ErrorView />);
+  });
+});
```
Comment (w-gao) [inline src/views/Error/index.test.js]: EOF

File: src/views/Home/index.test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import HomeView from './index.view';
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Testing Home View', () => {
+  it('Home Page renders without crashing', () => {
+    shallow(<HomeView />);
+  });
+});
```
Comment (w-gao) [inline src/views/Home/index.test.js]: EOF

File: src/views/Portal/index.test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import PortalView from './index.view';
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Testing Portal View', () => {
+  it('Portal Page renders without crashing', () => {
+    shallow(<PortalView />);
+  });
+});
```
Comment (w-gao) [inline src/views/Portal/index.test.js]: EOF

File: src/views/Team/index.test.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+import Adapter from 'enzyme-adapter-react-16';
+import TeamView from './index.view';
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('Testing Team View', () => {
+  it('Team Page renders without crashing', () => {
+    shallow(<TeamView />);
+  });
+});
```
Comment (w-gao) [inline src/views/Team/index.test.js]: EOF

**Review by w-gao — COMMENTED** [review]
I have a few comments (see below), but otherwise this looks good to me. Thanks for adding the tests!

**Review by Alperdec — APPROVED** [review]
looks good! thanks Krishna

**Pull request merged**